gen/suites/reward/main.go

```diff
@@ -24,4 +27,120 @@ func main() {
 			TipsetFunc: minersAwardedNoPremiums,
 		},
 	)
+
+	g.Group("penalties",
+		&VectorDef{
+			Metadata: &Metadata{
+				ID:      "penalize-inexistent-sender-multiple-miners",
+				Version: "v1",
+				Desc:    "verifies that a miner including a message from an inexistent sender is penalized; only the first miner is penalized",
+				Comment: "https://github.com/filecoin-project/lotus/issues/3491",
+			},
+			TipsetFunc: minerPenalized(3, func(v *TipsetVectorBuilder) {
+				v.StagedMessages.SetDefaults(GasLimit(1_000_000_000), GasPremium(0), GasFeeCap(200))
+
+				from := MustNewBLSAddr(1000)                // random
```

Comment:
where / how do we specify initial balance of this sending actor?

Reply:
This is a deliberately inexistent actor for the purposes of this test case.